threddsclient/__init__.py

```diff
@@ -1,3 +1,3 @@
 from .client import download_urls, opendap_urls, read_url, read_xml, crawl
 
-__version__ = '0.4.3'
+__version__ = '0.4.4'
```

Comment:
I would use `$ bump2version patch` to ensure this value changes in `setup.cfg` as well.

Reply:
I agree.

Reply:
This kind of config can be defined to let `bump2version` generate the changes entry automatically as well: 
https://github.com/Ouranosinc/Magpie/blob/master/setup.cfg#L1-L32